U/UserNSSandbox/build_tarballs.jl

```diff
@@ -1,17 +1,17 @@
 using BinaryBuilder
 
 name = "UserNSSandbox"
-version = v"2022.01.06"
+version = v"2022.01.20"
 
 # Collection of sources required to complete build
 sources = [
     GitSource("https://github.com/staticfloat/Sandbox.jl",
-              "9bd66da515893f1997feb51d228bcb3dc1d68771"),
+              "ed87f079edb1cef60c32ad5fbfbcd38c70a1314a"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd ${WORKSPACE}/srcdir/Sandbox.jl/deps
+cd ${WORKSPACE}/srcdir/Sandbox.jl-cac5c095dfe27fa774bdad65957d45f476b220680c9a2038c0dd2c25cdd0b3e6/deps
```

Comment:
`Sandbox.jl-*` to avoid having to update the script for the next version bump

Reply:
Good call, done!